libexec/rbenv-version-file-read

```diff
@@ -14,6 +14,19 @@ if [ -e "$VERSION_FILE" ]; then
   if [ "$version" = ".." ] || [[ $version == */* ]]; then
     echo "rbenv: invalid version in \`$VERSION_FILE'" >&2
   elif [ -n "$version" ]; then
+    if [[ "$VERSION_FILE" == *\.ruby-version ]]; then
+      variant_file="${VERSION_FILE/\.ruby-version/.ruby-variant}"
+    fi
+
+    if [[ -e "$variant_file" ]]; then
+      variant_words=( $(cut -b 1-1024 "$variant_file") )
```

Comment:
I think it's better if you use `cat "$variant_file"` here instead of `cut`. Using cat is simpler, easier to read, faster.

I see that the original code also used cut but I don't think they have a good reason to do that.